examples/position-pid-control/robot.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+# ----------------------------------------------------------------------------
+# Copyright (c) 2017-2018 FIRST. All Rights Reserved.
+# Open Source Software - may be modified and shared by FRC teams. The code
+# must be accompanied by the FIRST BSD license file in the root directory of
+# the project.
+# ----------------------------------------------------------------------------
+
+import rev
+import wpilib
+from wpilib.drive import DifferentialDrive
+
+
+class Robot(wpilib.TimedRobot):
+    def robotInit(self):
+        # Create motors
+        self.motor = rev.CANSparkMax(0, rev.MotorType.kBrushless)
+
+        # You must call getPIDController() on an existing CANSparkMax or
+        # SparkMax object to fully use PID functionality
+        self.pidController = self.motor.getPIDController()
+
+        # Instantiate built-in encoder to display position
+        self.encoder = self.motor.getEncoder()
+
+        # PID Coefficents and Controller Output Range
+        self.coeff = {"p": 0.1, "i": 1e-4, "d": 0, "iz": 0, "ff": 0}
```

Comment:
The integral zone is not a coefficient...

REV didn't write their examples like this either. Is there a particular reason why you didn't just copy and paste the existing Python code for this?

Reply:
Apologies for the errors. I didn't think twice after finishing them. Thanks for the feedback.